pre-docs/writing_tasks.md

```diff
@@ -628,25 +624,10 @@ There are two options for defining a task parameter as `Sensitive`:
 }
 ```
 
-2) Wrap the parameter with `Sensitive` in the `type` property within the JSON metadata: 
-
-```
-{
-  "description": "Task has a sensitive property denoted by its type",
-  "input_method": "stdin",
-  "parameters": {
-    "password": {
-      "description": "The password",
-      "type": "Sensitive[String[1]]",
-    }
-  }
-}
-```
-
-When a parameter is `Sensitive` the only expectation, currently, is that the parameter
-will not be logged. No other precautions are taken.
+When a parameter is `sensitive` the only expectation, currently, is that the parameter
+will not be logged (except when passing `--debug`). No other precautions are taken.
 
-It is up to the task author to implement their tasks in such a way that `Sensitive`
+It is up to the task author to implement their tasks in such a way that `sensitive`
 parameters are not exposed or logged. Please be mindful when coding your tasks!
```

Comment:
Can you also remove line 706/687, the entry about `Sensitive`? I think that was added accidentally, and doesn't work.